scss/_variables-dark.scss

```diff
@@ -124,10 +124,11 @@ $form-select-disabled-indicator-dark:   escape-svg(url("data:image/svg+xml,<svg
 // Boosted mod: no $form-switch-bg-image-dark
 
 // scss-docs-start form-validation-colors-dark
-$form-valid-color-dark:                 var(--#{$prefix}success-text-emphasis) !default; // Boosted mod: instead of `$green-300`
-$form-valid-border-color-dark:          var(--#{$prefix}success) !default; // Boosted mod: instead of `$green-300`
-$form-invalid-color-dark:               var(--#{$prefix}danger-text-emphasis) !default; // Boosted mod: instead of `$red-300`
-$form-invalid-border-color-dark:        var(--#{$prefix}danger) !default; // Boosted mod: instead of `$red-300`
+$form-valid-color-dark:                           var(--#{$prefix}success-text-emphasis) !default; // Boosted mod: instead of `$green-300`
+$form-valid-border-color-dark:                    var(--#{$prefix}success) !default; // Boosted mod: instead of `$green-300`
+$form-invalid-color-dark:                         var(--#{$prefix}danger-text-emphasis) !default; // Boosted mod: instead of `$red-300`
+$form-invalid-border-color-dark:                  var(--#{$prefix}danger) !default; // Boosted mod: instead of `$red-300`
+$form-switch-unchecked-invalid-border-color-dark: var(--#{$prefix}danger) !default; // Boosted mod
```

Comment:
This one should probably be set to `$form-invalid-border-color-dark` to ensure consistency within the framework; and it still can be overridden by users if needed.